snapcraft/integrations/travis.py

```diff
@@ -218,9 +218,8 @@ def enable():
             'skip_cleanup': True,
             'provider': 'script',
             'script': (
-                'docker run -v $(pwd):$(pwd) -t ubuntu:xenial sh -c '
-                '"apt update -qq && apt install snapcraft -y && cd $(pwd) && '
-                'snapcraft && snapcraft push *.snap --release edge"'),
+                'docker run -e TERM -v $PWD:$PWD -w $PWD -t snapcore/snapcraft'
+                ' sh -c "snapcraft && snapcraft push *.snap --release edge"'),
```

Comment:
@3v1n0 I think this will also need apt update and apt upgrade, because the snapcore/snapcraft image is not being updated: https://bugs.launchpad.net/snapcraft/+bug/1639112